types/canvasjs/index.d.ts

```diff
@@ -1,992 +1,1690 @@
-// Type definitions for CanvasJS v1.5.1
+// Type definitions for CanvasJS 1.9
 // Project: http://canvasjs.com/
-// Definitions by: Mark Overholt <https://github.com/mover5>
+// Definitions by: ShuYin Zhang  <https://github.com/brutalimp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace CanvasJS {
+export namespace CanvasJS {
```

Comment:
So, in the test file, i should  remove import statement and add `/// <reference types="canvasjs" />` , then in the .d.ts file change the `export ` to `declare`, at the end of the file add `export = CanvasJS;` and `export as namespace CanvasJS;`? if i do so, how should import this namespace in my ng2 project code, just `import "canvasjs"` or something else?

i really don't understand when should `declare` or `export` a `namespace` or `module`? those CMD,AMD and Typescript things  are really confusing.